game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -11,11 +13,12 @@ import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue}
 class GameThread(scoreThread: ScoreThread, teamIDs: List[Int]) extends Runnable {
   val MillisecondsPerTick = 16
 
+  final val actionQueue = new LinkedBlockingQueue[ActionQueryWithId]()
+  final val actionMap = new HashMap[Int, HashMap[Int, List[Action]]]()
```

Comment:
Can we add a comment to say what it means? Hard to tell looking only at the signature.
Or maybe a rename? `gameToActions`